food/ios/App/Detail/DetailViewController.swift

```diff
@@ -31,9 +32,32 @@ extension DetailViewController: UICollectionViewDataSource {
   }
 }
 
+extension DetailViewController: LightboxControllerDismissalDelegate {
+  func lightboxControllerWillDismiss(_ controller: LightboxController) {
+    self.collectionView.scrollToItem(at: IndexPath(item: controller.currentPage, section: 0),
+                                     at: .centeredHorizontally,
+                                     animated: false)
+  }
+}
+
 extension DetailViewController: UICollectionViewDelegate {
 
   func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+    guard let post = place.post else { return }
+
+    let images : [LightboxImage] = post.images.map {
+      LightboxImage(
+        imageURL: $0.url!,
+        text: $0.caption!
```

Comment:
Captions are optional, not required by our CMS.
Will include `credit` here as well.